app/src/main/java/de/westnordost/streetcomplete/data/meta/CountryInfo.kt

```diff
@@ -0,0 +1,116 @@
+package de.westnordost.streetcomplete.data.meta
+
+import de.westnordost.streetcomplete.osm.LengthUnit
+import de.westnordost.streetcomplete.quests.max_speed.SpeedMeasurementUnit
+import de.westnordost.streetcomplete.quests.max_weight.WeightMeasurementUnit
+import kotlinx.serialization.Serializable
+import java.util.Locale
+
+@Serializable
+data class IncompleteCountryInfo(
+    // this value is not defined in the yaml file but it is the ISO language code part of the file name!
+    // i.e. US for US-TX.yml
+    val countryCode: String,
+
+    // sorted alphabetically for better overview
+    val additionalStreetsignLanguages: List<String> = emptyList(),
+    val additionalValidHousenumberRegex: String? = null,
+    val advisorySpeedLimitSignStyle: String? = null,
+    val atmOperators: List<String> = emptyList(),
+    val centerLineStyle: String? = null,
+    val chargingStationOperators: List<String> = emptyList(),
+    val clothesContainerOperators: List<String> = emptyList(),
+    val edgeLineStyle: String? = null,
+    val firstDayOfWorkweek: String? = null,
+    val hasAdvisorySpeedLimitSign: Boolean? = null,
+    val hasBiWeeklyAlternateSideParkingSign: Boolean? = null,
+    val hasCenterLeftTurnLane: Boolean? = null,
+    val hasDailyAlternateSideParkingSign: Boolean? = null,
+    val hasLivingStreet: Boolean? = null,
+    val hasNoStandingSign: Boolean? = null,
+    val hasSlowZone: Boolean? = null,
+    val isLeftHandTraffic: Boolean? = null,
+    val isUsuallyAnyGlassRecyclableInContainers: Boolean? = null,
+    val lengthUnits: List<LengthUnit> = emptyList(),
+    val livingStreetSignStyle: String? = null,
+    val mobileCountryCode: Int? = null,
+    val noParkingLineStyle: String? = null,
+    val noParkingSignStyle: String? = null,
+    val noStandingLineStyle: String? = null,
+    val noStandingSignStyle: String? = null,
+    val noStoppingLineStyle: String? = null,
+    val noStoppingSignStyle: String? = null,
+    val officialLanguages: List<String> = emptyList(),
+    val orchardProduces: List<String> = emptyList(),
+    val popularReligions: List<String> = emptyList(),
+    val popularSports: List<String> = emptyList(),
+    val regularShoppingDays: Int? = null,
+    val roofsAreUsuallyFlat: Boolean? = null,
+    val slowZoneLabelPosition: String? = null,
+    val slowZoneLabelText: String? = null,
+    val speedUnits: List<SpeedMeasurementUnit> = emptyList(),
+    val weightLimitUnits: List<WeightMeasurementUnit> = emptyList(),
```

Comment:
Any types used in CountryInfo should probably be moved to this directory or even into this file.

Reply:
Fixed in [`3a7a88a`](https://github.com/streetcomplete/StreetComplete/pull/3715/commits/3a7a88a248ffe708666c263e708baf6a333a25f6).